freyja_examples/src/ref_to_point.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/python
+
```

Comment:
Consider adding a block-comment here to briefly say what the file is for: converts to `geometry_msgs` type for visualisation purposes (rqt_) only.

Reply:
This was accidentally uploaded in this merge. I wasn't planning on adding it, I have removed it from this merge. However if you think it would be useful to include I will bring it back.